yarn.lock

```diff
@@ -5973,6 +5973,14 @@ svelte-preprocess@^6.0.3:
   resolved "https://registry.yarnpkg.com/svelte-preprocess/-/svelte-preprocess-6.0.3.tgz#fdc1f9dc41b6f22bf8b1f059e9f21eaaae181eeb"
   integrity sha512-PLG2k05qHdhmRG7zR/dyo5qKvakhm8IJ+hD2eFRQmMLHp7X3eJnjeupUtvuRpbNiF31RjVw45W+abDwHEmP5OA==
 
+svelte-vega@^1.2.0:
```

Comment:
That's weird. Probably shouldn't be here.

Reply:
hmm thats's what I get when I run `yarn` on the monorepo.

Reply:
Seems to indicate that we don't use the dependency from this repo somehow.